src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -29,6 +29,10 @@ export class MultiSelectComponent<T> implements OnInit, ControlValueAccessor {
   @Input() public label: string
   @Input() public placeholder?: string
 
+  @Input() public hasError?: boolean
```

Comment:
This attribute should be deleted. Refer to my comment on `TextInputComponent`.

Reply:
Fixed